### ISSUE TITLE
[v2] issues with redux (lodash-es) after beta 97

### ISSUE THREAD

**Issue by cpboyd** [gatsbyjs/gatsby#7333]
## Description

Unable to run `gatsby develop` on projects using `lodash-es` (which is used by `react-redux`).

### Steps to reproduce

Try to run `gatsby develop` on a project with `react-redux` on the current build.

### Expected result

Build should be successful.

### Actual result

```
error UNHANDLED REJECTION

  Error: .../node_modules/lodash-es/isPlainObject.js:1
  (function (exports, require, module, __filename, __dirname) { import baseGetTag from './_baseGetTag.js';
                                                                       ^^^^^^^^^^
  SyntaxError: Unexpected identifier

  - loader.js:657 Module._compile
    internal/modules/cjs/loader.js:657:28

  - loader.js:700 Object.Module._extensions..js
    internal/modules/cjs/loader.js:700:10

  - loader.js:599 Module.load
    internal/modules/cjs/loader.js:599:32

  - loader.js:538 tryModuleLoad
    internal/modules/cjs/loader.js:538:12

  - loader.js:530 Function.Module._load
    internal/modules/cjs/loader.js:530:3

  - loader.js:637 Module.require
    internal/modules/cjs/loader.js:637:17

  - helpers.js:20 require
    internal/modules/cjs/helpers.js:20:18

  - render-page.js:3 webpackUniversalModuleDefinition
   .../public/render-page.js:3:1321

  - render-page.js:10 Object.<anonymous>
   .../public/render-page.js:10:3

  - loader.js:689 Module._compile
    internal/modules/cjs/loader.js:689:30

  - loader.js:700 Object.Module._extensions..js
    internal/modules/cjs/loader.js:700:10

  - loader.js:599 Module.load
    internal/modules/cjs/loader.js:599:32

  - loader.js:538 tryModuleLoad
    internal/modules/cjs/loader.js:538:12
```

### Environment

```

  System:
    OS: macOS High Sierra 10.13.6
    CPU: x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.8.0 - /usr/local/bin/node
    Yarn: 1.9.4 - /usr/local/bin/yarn
    npm: 6.3.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 68.0.3440.106
    Safari: 11.1.2
  npmPackages:
    gatsby: next=> 2.0.0-beta.100
    gatsby-image: next => 2.0.0-beta.7 
    gatsby-plugin-google-analytics: next => 2.0.0-beta.5 
    gatsby-plugin-jss: next => 2.0.2-beta.3 
    gatsby-plugin-manifest: next => 2.0.2-beta.5 
    gatsby-plugin-offline: next => 2.0.0-beta.9 
    gatsby-plugin-react-helmet: next => 3.0.0-beta.4 
    gatsby-plugin-sharp: next => 2.0.0-beta.7 
    gatsby-plugin-typescript: next => 2.0.0-beta.9 
    gatsby-plugin-typography: next => 2.2.0-beta.3 
    gatsby-remark-external-links: ^0.0.4 => 0.0.4 
    gatsby-remark-images: next => 2.0.1-beta.10 
    gatsby-remark-responsive-iframe: next => 2.0.0-beta.3 
    gatsby-source-filesystem: next => 2.0.1-beta.10 
    gatsby-transformer-json: next => 2.1.1-beta.5 
    gatsby-transformer-remark: next => 2.1.1-beta.6 
    gatsby-transformer-sharp: next => 2.1.1-beta.7 
  npmGlobalPackages:
    gatsby-cli: 1.1.58
```

### File contents (if changed)

`gatsby-config.js`: N/A 
`package.json`: N/A 
`gatsby-node.js`: N/A 
`gatsby-browser.js`: N/A 
`gatsby-ssr.js`: N/A 

Will try to get a repro up soon.

**Comment by cpboyd** [comment]
I checked for other issues, but didn't see that PR.

Cheers!

**Comment by m-allanson** [comment]
Fixed in `gatsby@2.0.0-beta.102`
